src/webots/gui/WbSimulationView.cpp

```diff
@@ -555,7 +559,7 @@ void WbSimulationView::startVideoCapture(const QString &fileName, int codec, int
 
 void WbSimulationView::stopVideoCapture(bool canceled) {
   WbVideoRecorder::instance()->stopRecording(canceled);
-  restoreFastModeIfNecessary();
+  restore3dViewIfNecessary();
```

Comment:
Currently there are for example this function `restore3dViewIfNecessary` and another one `is3DViewShown`.
We should be consistent in the variable and function name and always use `3D` or `3d`.
it would probably be better to always use `3D`.

Reply:
You right, I missed changing this one from `3d` to `3D`